.github/workflows/analyze.yml

```
@@ -118,7 +118,17 @@ jobs:
+      - name: Keep heavy SDKs off the startup import chain (lazy-import contract)
+        # Merged production mode imports the app tree serially before any port
+        # binds, so a module-scope `import openai` (etc.) slows EVERY launch,
+        # silently — reference incident: #1496 cut main_server import to ~0.6s,
+        # openai 2.x types growth quietly ate it back to ~2.1s within 6 weeks.
+        # Banned-at-module-scope list (openai/anthropic/bs4/bilibili_api/
+        # google.genai/translatepy/dashscope/...) lives in the script; the
+        # sanctioned pattern is in-function import + utils/module_warmup.py
+        # background pre-import after ready. `# noqa: STARTUP_LAZY_IMPORT`
+        # opts out with justification.
+        run: python scripts/check_startup_import_lazy.py
     name: Python conventions (i18n-sync + docs + api-trailing-slash×2 + module-layering + docstring-cjk)
```

actionlint (1.7.12)

[error] key "name" is duplicated in "python-lint" job. previously defined at line:27,col:5

(syntax-check)